core/primitives/block.cpp

```diff
@@ -5,17 +5,4 @@
 
 #include "primitives/block.hpp"
 
-namespace kagome::primitives {
-
-  Block::Block(BlockHeader header, std::vector<Extrinsic> extrinsics)
-      : header_(std::move(header)), extrinsics_(std::move(extrinsics)) {}
-
-  const BlockHeader &Block::header() const {
-    return header_;
-  }
-
-  const std::vector<Extrinsic> &Block::extrinsics() const {
-    return extrinsics_;
-  }
-
-}  // namespace kagome::primitives
+namespace kagome::primitives {}  // namespace kagome::primitives
```

Comment:
Are cpp files for structures needed at all?

Reply:
They will probably be needed when implementing 3-rd phase of task, implementing encode/decode functions.

Reply:
if no, I will remove them.